core/controller/src/main/resources/apiv1swagger.json

```diff
@@ -1889,21 +1889,16 @@
                         "blackbox",
                         "java",
                         "java:default",
-                        "nodejs:6",
                         "nodejs:8",
                         "nodejs:10",
                         "nodejs:default",
-                        "php:7.1",
-                        "php:7.2",
                         "php:7.3",
```

Comment:
Noticed we don’t have php:default let’s add it :) and the other runtimes that should have it too.